spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,179 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> empty/input.scss
+a {b: split("")}
+
+<===> empty/output.css
+a {
+  b: [""];
```

Comment:
According to the spec as written, an empty input string should always return an empty array, since `string.length($string)` will be `0`.

Reply:
fixed this